src/python/pants/backend/jvm/tasks/jvm_compile/compile_context.py

```diff
@@ -8,18 +8,20 @@
 import zipfile
 from contextlib import contextmanager
 
+from pants.backend.jvm.targets.jar_library import JarLibrary
 from pants.build_graph.aliased_target import AliasTarget
 from pants.build_graph.target import Target
 from pants.util.contextutil import open_zip
 
 
 def _resolve_strict_dependencies(target):
```

Comment:
Both `strict_deps` and `exports` are currently jvm `*_library`-target-only concepts. The question is whether they should work with more target types in the future.